02-set-up.Rmd

```diff
@@ -125,32 +109,43 @@ server management. Fortunately there are many tools designed to ease monitoring
 - On Linux, the shell command `top` displays key resource use figures for most
 distributions. `htop` and Gnome's **System Monitor** (`gnome-system-monitor`, see figure \@ref(fig:2-1)) are more refined alternatives which use command-line and graphical user interfaces respectively.
 A number of options such as `nethogs` monitor internet usage.
-- On Windows the **Task Manager** provides key information on RAM and CPU use by
-process. This can be started in modern Windows versions by typing `Ctrl-Alt-Del` or by clicking the task bar and 'Start Task Manager'.
 - On Mac the **Activity Monitor** provides similar functionality. This can be initiated
 from the Utilities folder in Launchpad.
+- On Windows the **Task Manager** provides key information on RAM and CPU use by
+process. This can be started in modern Windows versions by typing `Ctrl-Alt-Del` or by clicking the task bar and 'Start Task Manager'.
 
 #### Exercises {-}
 
 1. What is the exact version of your computer's operating system?
-2. Start an activity monitor then type and execute the following code. How do the results on your system compare to those presented in Figure \@ref(fig:2-1)? 
+2. Start an activity monitor then execute the following code chunk.
+In it `lapply()` (or its parallel version `mclapply()`) is used to *apply* a function, `median()`, over every column in the data frame object
+`X` (see Section \@ref(the-apply-family) for more on the 'apply family' of 
+functions).[The
+reason this works is that a data frame is really a collection of list objects.
```

Comment:
Good point - fixed here: https://github.com/csgillespie/efficientR/pull/167/commits/81ae41b92eb3cda092a040a7e9ebf7478b4d025d

Ideally I'd like to link to another part of the book that expands on this but couldn't find it. Any ideas where we could direct the interested people to? Sure there are plenty of good online resources explaining this too.